spec.emu

```diff
@@ -417,15 +420,12 @@ contributors: Luca Casonato, Guy Bedford, Nicolò Ribaudo
             </tr>
             <tr>
               <td>
-                <ins>GetModuleSource(): either a normal completion containing an Object or a throw completion</ins>
+                <ins>GetModuleSource()</ins>
               </td>
               <td>
-                <p><ins>Returns a completion for the ModuleSource Object corresponding to this source Module Record's source phase (<emu-xref href="#sec-module-source-objects"></emu-xref>), if it has been requested.</ins></p>
-
-                <emu-note type="editor">
-                  <ins>When called on the same Module Record, the Object for the normal completion of GetModuleSource must be an idempotent unique return value.</ins>
-                  <ins>For a normal completion, the Object is expected to be a subclass of %AbstractModuleSource%, with an internal slot `[[ModuleSourceRecord]]`.</ins>
-                </emu-note>
+                <p><ins>It returns either a normal completion completion for the ModuleSource Object corresponding to this source Module Record's source phase (<emu-xref href="#sec-module-source-objects"></emu-xref>), or a throw completion.</ins></p>
+                <ins>When called multiple times on the same Module Record, if GetModuleSource() returns a normal completion it must always return a normal completion containing the same object.</ins>
+                <ins>The returned object should be a subclass of %AbstractModuleSource%, and it must have an internal slot [[ModuleSourceRecord]].</ins>
```

Comment:
There are not just notes, but normative requirements. I used "should" for the subclassing bit because as already discussed it's not really enforceable.